src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/MrdTestDataBuilder.kt

```diff
@@ -183,6 +185,28 @@ class MrdTestDataBuilder {
             riskOfSeriousHarmLevel = "High",
             sentenceDate = LocalDate.now(),
           ),
+          ppudOffender = PpudOffender(
+            id = "12345678",
+            croOtherNumber = "1234",
+            dateOfBirth = "14 09 1990",
```

Comment:
Dates will be returned as ISO Local Date format i.e. dateOfBirth 1990-09-14, dateOfSentence 2006-01-01